src/panoptoindexconnector/connector_config.py

```diff
@@ -100,6 +100,11 @@ def panopto_user_group_mapping(self):
     def panopto_id_provider_instance_name(self):
         return self._yaml_config['panopto_id_provider_instance_name']
 
+    @property
+    def panopto_id_provider_is_unified(self):
+        # ensures that configuration value is parsed correctly
+        return str(self._yaml_config.get('panopto_id_provider_is_unified')).lower() == 'true'
```

Comment:
🍎- using `.get` means this will return `None` if the key is not present, which will crash on the `.lower()` call. Think about what the default should be I'd say if we are not going to require this field to be filled in and modify to ge `.get('...', <default>)....`

Also -- doesn't the yaml config return boolean values as boolean, not string? If so, looks like this would crash anyway... (test below)
![image](https://user-images.githubusercontent.com/4721718/195151108-fe16b0e6-e527-4379-9f52-f9d3f8e9b118.png)